docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
+schema: Schema.yaml
```

Comment:
Is `Schema.yaml` a valid example of a schema file name, now that we've standardized on JSON schema?